.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:10.18.0
+    - image: circleci/node:15
```

Comment:
it's better to use LTS which is 14 currently. The support of node 15 will end this June. https://nodejs.org/en/about/releases/

Reply:
I updated the image to have node 7 due to the different behavior `npm pack` (which provided by node 15 by default). It is just the runtime for the build process, does not affect the published package.